src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -1081,6 +1081,7 @@ public static partial class JsonMetadataServices
         public static System.Text.Json.Serialization.JsonConverter<System.Text.Json.Nodes.JsonNode> JsonNodeConverter { get { throw null; } }
         public static System.Text.Json.Serialization.JsonConverter<System.Text.Json.Nodes.JsonObject> JsonObjectConverter { get { throw null; } }
         public static System.Text.Json.Serialization.JsonConverter<System.Text.Json.Nodes.JsonValue> JsonValueConverter { get { throw null; } }
+        public static void MakeReadOnly(System.Text.Json.Serialization.Metadata.JsonTypeInfo jsonTypeInfo) { throw null; }
```

Comment:
This is the new API addition: adding directly to `JsonTypeInfo` would invite scrutiny w.r.t. naming and shape, so proxying via the `EditorBrowsable.Never` APIs of the `JsonMetadataServices` class.